qiime2/sdk/tests/test_method.py

```diff
@@ -523,9 +516,7 @@ def test_docstring(self):
 
 Returns
 -------
-out: Mapping
-    No description available
-
+out : Mapping\
```

Comment:
There should be a trailing empty line (I'm not 100% if that's the spec, but that's our convention that we use):
```
"""Docstring start
.... stuff ....
   <- single empty line
"""
```